mock-core-configs/etc/mock/templates/opensuse-tumbleweed.tpl

```diff
@@ -1,4 +1,4 @@
-config_opts['chroot_setup_cmd'] = 'install patterns-devel-base-devel_rpm_build'
+config_opts['chroot_setup_cmd'] = 'install patterns-devel-base-devel_rpm_build openSUSE-release-ftp rpm-repos-openSUSE-Tumbleweed'
```

Comment:
This should be `openSUSE-release` instead of `openSUSE-release-ftp`. Otherwise this is good.